CHANGES.txt

```diff
@@ -15,6 +15,13 @@ the appropriate section of the docs.
 Breaking Changes
 ================
 
+ - The default value of the setting ``auth.host_based.enabled`` (``false``) is
```

Comment:
Not sure - but maybe we should write out the implications here a bit more explicitly - in that connections from remote hosts are no longer possible with the default configuration (unless enterprise is off), and maybe even mention explicitly that this is always the case for docker as it's always remote?

@joemoe ^
do you think it might make sense to do any up-front communication that includes ways that user can "future proof" (their test jobs, docker-compose files, etc.), so that things don't start breaking if they're using `crate:latest` instead of a nailed version?

Reply:
I am not sure if we should write about Docker implication in the changelog. A blog post about the upcoming release, once it is testing, could be much more suitable place to communicate these changes.

Reply:
> I am not sure if we should write about Docker implication in the changelog

I'm also not sure, but would it hurt to have it here?
If it's not explicitly mentioned the worst thing that can happen is that someone reads it as it is and isn't able to connect the dots and therefore gets angry when his docker setup breaks.

If it's there the worst thing that can happen is that somebody reads it and thinks "duh, that's obvious" (or because it's more text they just skip reading it altogether because tl;dr).

I'm thinking of people here who only skim through the release notes. They might pick up "docker" because they realize that might be relevant to them, but they might not pick up authorization because they currently don't use it.

Blog post would be a good addition.
Anyway, just sharing my thoughts, no too strong opinion here.

Reply:
Oh btw - the password auth feature itself is not mentioned in the changes?

Reply:
> the password auth feature itself is not mentioned in the changes?
@mfussenegger good catch!

Reply:
added a proposal for the breaking change in new fixup commit